src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -50,6 +50,12 @@ func main() {
 }
 @end
 
+@private scopeArg(method)
+  @join scope : context.getAuthScopes(method) on ""
```

Comment:
Maybe get rid of the loop for clarity/consistency with getAuthScopes returning a singleton list at most.